ghengin/ghengin-camera/Ghengin/Camera.hs

```diff
@@ -33,10 +33,12 @@ data Camera (view_field :: Symbol) (proj_field :: Symbol)
            deriving anyclass Block
 
 -- | A default camera looking at (0, 0, 0) using a perspective projection.
+--
+-- +X right, +Y down, +Z forward
 defaultCamera :: Camera view_field proj_field
 defaultCamera = Camera
-  { view = unTransform $ lookAt (vec3 0 0 0) (vec3 0 0 1) (vec3 0 1 0)
-  , proj = unTransform $ perspective @Int 45 0.1 1000 640 480
+  { view = unTransform $ lookAtRH (vec3 0 0 0) (vec3 0 0 1) (vec3 0 (-1) 0)
+  , proj = unTransform $ reverseDepthRH 45 0.1 640 480
```

Comment:
BTW, projections use radians in FoV.
45 is pi/4 if you really want such a narrow view.
The full 90deg vertical fov would be pi/2 (it does the half-angle thing by itself).